doc/code_snippets/snippets/config/instances.enabled/isolated_mode/config.yaml

```diff
@@ -0,0 +1,10 @@
+groups:
+  g:
+    replicasets:
+      r:
+        instances:
+          i-001: {}
+          i-002: {}
+          i-003: {}
+          i-004:
+            isolated: true
```

Comment:
Nit: GitHub shows a red circle, when there is no newline symbol at the end of the file.

![image](https://github.com/user-attachments/assets/1e91267b-8059-4e46-b821-01995d61e7ae)